e2e/test/scenarios/admin/datamodel/editor.cy.spec.js

```diff
@@ -633,7 +633,7 @@ const getFieldSection = fieldName => {
 };
 
 const moveField = (fieldIndex, deltaY) => {
-  cy.get(".Grabber").eq(fieldIndex).trigger("mousedown", 0, 0);
+  cy.get(".Grabber").eq(fieldIndex).trigger("mousedown", 0, 0, { force: true });
```

Comment:
Interestingly,[ this just fails](https://www.deploysentinel.com/ci/runs/6481ca4014a47e6dd93d5a65) on the latest commit https://github.com/metabase/metabase/pull/31210/commits/19cc26b0be3c11a28d2fae37163d41a7662afff2, which only added unit tests 🤔 . Maybe it has something to do with rebasing off `master` too.

But I think this is justified since other place that `trigger("mousedown", xxx)` all uses `force: true`
![Screenshot 2023-06-08 at 9 24 43 PM](https://github.com/metabase/metabase/assets/1937582/e328f8ca-92b7-44ff-b1ca-e34d6a9431e4)